.pre-commit-config.yaml

```diff
@@ -0,0 +1,14 @@
+repos:
+-   repo: https://github.com/psf/black
+    rev: 21.5b1
+    hooks:
+    -   id: black
+        exclude: |
+            (?x)^(
+                (tardis/model
+                | tardis/montecarlo).*
+                | setup.py
+                | docs/conf.py
+                | ez_setup.py
+                | ah_bootstrap.py
+            )$
```

Comment:
Why are the above files excluded from black?

Reply:
I don't know, they are excluded in the existing Black [config](https://github.com/tardis-sn/tardis/blob/351cc8f7b546f9eddf94608dbba2908ed333281b/pyproject.toml#L4) so I just copied them to the pre-commit Black config.